doc/devel/index.rst

```diff
@@ -5,72 +5,140 @@ Contribute
-        :color: primary
+         :octicon:`question;1em;sd-text-info` :ref:`get_connected`
 
-        Report a bug
+         :octicon:`issue-opened;1em;sd-text-info` :ref:`new_contributors`
 
-.. rst-class:: sd-d-inline-block
+         :octicon:`git-pull-request;1em;sd-text-info` :ref:`managing_issues_prs`
 
-    .. button-ref:: request-a-new-feature
-        :class: sd-fs-6
-        :color: primary
+         :octicon:`codespaces;1em;sd-text-info` placeholder for codespaces link
 
-        Request a feature
+      .. grid-item::
 
-.. rst-class:: sd-d-inline-block
+         .. grid:: 1
+            :gutter: 1
+            :class-row: sd-fs-5
 
-    .. button-ref:: contributing-code
-        :class: sd-fs-6
-        :color: primary
+            .. grid-item-card::
+               :link: request-a-new-feature
+               :link-type: ref
 
-        Contribute code
+               :octicon:`light-bulb;1em;sd-text-info` Request new feature
 
-.. rst-class:: sd-d-inline-block
+            .. grid-item-card::
+               :link: submitting-a-bug-report
+               :link-type: ref
 
-    .. button-ref:: documenting-matplotlib
-        :class: sd-fs-6
-        :color: primary
+               :octicon:`bug;1em;sd-text-info` Submit bug report
 
-        Write documentation
+            .. grid-item-card::
+               :link: contributing-code
+               :link-type: ref
 
-.. toctree::
-   :maxdepth: 2
-
-   contribute.rst
-   triage.rst
-   development_setup.rst
-   development_workflow.rst
-   testing.rst
-   document.rst
-   style_guide.rst
-   coding_guide.rst
-   release_guide.rst
-   dependencies.rst
-   min_dep_policy.rst
-   maintainer_workflow.rst
-   MEP/index
+               :octicon:`code;1em;sd-text-info` Contribute code
+
+            .. grid-item-card::
+               :link: documenting-matplotlib
+               :link-type: ref
+
+               :octicon:`note;1em;sd-text-info` Write documentation
+
+If you are new to contributing, we recommend that you first read our
+:ref:`contributing guide<contributing>`. If you are contributing code or
+documentation, please follow our guides for setting up and managing a
+:ref:`development environment and workflow<development_environment>`.
+For code, documentation, or triage, please follow the corresponding
+:ref:`contribution guidelines <contribution_guideline>`.
+
+
+.. _development_environment:
+
+Development workflow
+==============================
+
+.. grid:: 1 1 2 2
+
+   .. grid-item-card:: Install
+
+      .. toctree::
+         :maxdepth: 2
+
+         development_setup
+
+      .. toctree::
+         :maxdepth: 1
+
+         dependencies
+         Minimum versions <min_dep_policy>
+
+
+   .. grid-item-card:: Workflow
+
+      .. toctree::
+         :maxdepth: 2
+
+         development_workflow
+
+
+
+.. _contribution_guideline:
+
+Contribution guides
+===================
+
+.. grid:: 1 1 2 2
+
+   .. grid-item-card:: Code
+
+      | :ref:`coding_guidelines`
```

Comment:
Really small nit but on my screen the text for this link looks a bit smaller than for the two links below it, which just looks a bit odd.  I guess it’s because it’s a subsection title rather than a page title like the others.  Is there an easy way to override the size?